.github/workflows/gradle-wrapper-update.yml

```diff
@@ -0,0 +1,37 @@
+---
+name: Update Gradle wrapper scripts
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: 0 10 * * MON
```

Comment:
@timtebeek, any opinions on when and how often to run this workflow?

Reply:
We tend to release on wednesdays, so I'd say `TUE` would mean we're usually up to date by the time we release.